PROVESFlightControllerReference/test/unit-tests/test_PacketProcessor_Validator.cpp

```
@@ -0,0 +1,81 @@
+    Packet p{0u, 10u, 0xDEADBEEFu, {}};  // spi 0 valid
+    auto res = validatePacket(p, 10u, 5u);
+    EXPECT_EQ(res, PacketValidator::Status::Valid);
+}
+
+TEST(PacketValidatorTest, BypassNoOpOpCode) {
+    Packet p{0u, 10u, 0x01000000u, {}};  // bypass opcode
+    auto res = validatePacket(p, 10u, 5u);
+    EXPECT_EQ(res, PacketValidator::Status::Bypass);
+}
+
+TEST(PacketValidatorTest, BypassUartGetSeqNumOpCode) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)